CryptoPrices/Sources/Home/Sources/Home/MyCoins/MyCoinSection.swift

```diff
@@ -30,6 +32,13 @@ struct MyCoinSection: View {
                     ForEach(0..<10) {_ in
                         MyCoinItemView()
                             .padding(.trailing, 8.0)
+                            .onTapGesture {
```

Comment:
I think we also need to handle the `onTapGesture` to show the Coin Detail screen for the `TrendingCoinsSection` as there are coin item that can be selected too 🙏

Reply:
I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/56/commits/fad45b30c47233077c24d1567966fd76539f1aa6 🙏